app/helpers/application_helper.rb

```diff
@@ -1,6 +1,6 @@
 module ApplicationHelper
-  def full_title(page_title = " ")
-    base_title = t(".helpers.application.title")
+  def full_title(page_title = "")
+    base_title = t"helpers.application.ruby_on"
```

Comment:
sau t cách 1 spaces ra
```suggestion
    base_title = t "helpers.application.ruby_on"
```

Reply:
vâng e sửa ngay ạ !